syncd/VendorSai.cpp

```diff
@@ -4,6 +4,10 @@
 
 #include "swss/logger.h"
 
+#ifdef MDIO_ACCESS_USE_NPU
+#include "MdioIpcServer.h"
+#endif
+
```

Comment:
We could set VendorSai class shared_ptr in MdioIpcServer constructor. But we need to declare 4 virtual functions in SaiInterfaces class and implement the 4 functions in VendorSai class. Because there are other class with inheritance from SaiInterfaces class, the code change impact is much bigger than using 2 MdioIpcServer static functions here.

Reply:
yes, please declare those virtual functions

Reply:
virtual functions mdioRegRead/mdioRegWrite are declared in SaiInterfaces class and are overridden in VendorSai class